src/main/scala/com/codacy/client/bitbucket/v2/service/WorkspaceServices.scala

```diff
@@ -24,7 +24,7 @@ class WorkspaceServices(client: BitbucketClient) {
       username: String
   ): RequestResponse[Seq[User]] = {
     val encodedUsername = URLEncoder.encode(username, "UTF-8")
-    val baseRequestUrl = s"""${client.workspacesBaseUrl}/$encodedUsername/members?q=permission=$encodedUsername"""
+    val baseRequestUrl = s"""${client.workspacesBaseUrl}/$encodedUsername/members?q=permission="owner""""
```

Comment:
This should probably be encoded. Did you test it?

Reply:
👍